authapi/tests/test_organizations.py

```diff
@@ -544,6 +560,24 @@ def test_get_teams_for_organization(self):
 
         self.assertEqual(team['id'], team1.id)
 
+    def test_permission_teams_for_organization(self):
+        '''Any member of the organization should be able to see that
+        organization's teams.'''
+        org = SeedOrganization.objects.create()
+        url = reverse('seedorganization-teams-list', args=(org.pk,))
+
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
```

Comment:
Done in https://github.com/praekelt/seed-auth-api/pull/26/commits/c7cf82105abc741213bbb8a7f88c4a1407448ad3